RELEASE.md

```diff
@@ -0,0 +1,4 @@
+# Releasing
+
+1. Merge the release PR
+2. Run `npm publish`
```

Comment:
can we add https://github.com/googleapis/release-please#how-should-i-write-my-commits and general instructions to this doc?

Reply:
Sure I'll add the instructions

Reply:
I replaced this with a contributing guide, please let me know what you think